admin/featured.php

```diff
@@ -12,9 +12,19 @@
   $currencies = new currencies();
 
   $action = (isset($_GET['action']) ? $_GET['action'] : '');
+  if (!isset($_GET['reset_featured_sort_order'])) {
+    $reset_featured_sort_order = $_SESSION['featured_sort_order'];
+  }
+
+  zen_set_time_limit(900);
 
   if (zen_not_null($action)) {
     switch ($action) {
+      case 'set_featured_sort_order':
+        $_SESSION['featured_sort_order'] = $_GET['reset_featured_sort_order'];
+        $action='';
+        zen_redirect(zen_href_link(FILENAME_FEATURED, (isset($_GET['page']) && $_GET['page'] > 0 ? 'page=' . $_GET['page'] . '&' : '') . (isset($_GET['sID']) && $_GET['sID'] > 0 ? 'sID=' . $_GET['sID'] : '') . (isset($_GET['search']) && trim($_GET['search']) != ''  ? '&search=' . $_GET['search'] : '')));

```

Comment:
You're using `$_GET['sID']` here, but this file is for "Featured", and I think it should be `fID` instead.

Note the code from 6 lines further down:

```php
  zen_redirect(zen_href_link(FILENAME_FEATURED, (isset($_GET['page']) ? 'page=' . $_GET['page'] . '&' : '') . 'fID=' . $_GET['id'] . (isset($_GET['search']) ? '&search=' . $_GET['search'] : ''), 'NONSSL'));
```